### PR TITLE
Fix nil pointer dereference on call error

### PR DESCRIPTION
**Pull request by hevanto**
Fixes a panic for chargepoints that do not respond a valid errorDescription when issueing a faulty request.

### CODE REVIEW

File: ocppj/ocppj.go [hunk 1]
```diff
@@ -406,11 +406,15 @@ func (endpoint *Endpoint) ParseMessage(arr []interface{}, pendingRequestState Cl
 		}
 		rawErrorCode := arr[2].(string)
 		errorCode := ocpp.ErrorCode(rawErrorCode)
+		errorDescription := ""
+		if arr[3] != nil {
```
Comment (frzifus) [inline ocppj/ocppj.go:410]:
```suggestion
		if v, ok := arr[3].(string); ok {
			errorDescription = v
		}
```

Would it make sense to check the type at the same time?

Comment (hevanto) [reply ocppj/ocppj.go:410]: Absolutely, makes total sense.
I'll adjust it accordingly.

Comment (hevanto) [reply ocppj/ocppj.go]: Got a coverage drop now on the tests, but I don't see where it would come from ...

Comment (frzifus) [reply ocppj/ocppj.go]: you can use the go tool `cover` to see what is not covered. I guess there is no test case that contains a `string` value on `arr[3]`. As a result this condition is creating one more potential hit that is not tested - in sum then the test coverage drops.

```bash
go test -v -coverprofile cover.out
go tool cover -html=cover.out
```

Comment (lorenzodonini) [reply ocppj/ocppj.go]: If you look at the full codecov report, the coverage drop comes up in parts that weren't even affected by the changes. This happens quite a lot and it's always a negligible increase/drop in coverage. I'll investigate cause it bothers me, but that 0.02% drop is not a big deal.

Code looks good, thanks @hevanto for submitting the PR!

**Pull request merged**